### DISCUSSION TITLE
How to keep the Slingplayer on top

### DISCUSSION THREAD

**Discussion by lehmbarsingh** [GerryDazoo/SlingerPlayer#4] [General]
I would like to keep my slingplayer on top of everything while I am browsing the pc. Is there way to keep on top just like SMplayer has the option.

**Comment by GerryDazoo** [comment]
Not in the latest release. I'll add this to my list of requested enhancements.

**Comment by GerryDazoo** [comment]
give this a try. Check AoT (Always on Top).
[SlingerPlayer.zip](https://github.com/GerryDazoo/SlingerPlayer/files/10851231/SlingerPlayer.zip)

**Reply by lehmbarsingh** [reply]
wow ,That was fast .It works .Thanks .
